package/firmware/cypress-firmware/Makefile

```
@@ -10,13 +10,13 @@ include $(TOPDIR)/rules.mk
 UNPACK_CMD=unzip -q -p $(DL_DIR)/$(PKG_SOURCE) $(PKG_SOURCE_UNZIP) | gzip -dc | $(HOST_TAR) -C $(1) $(TAR_OPTIONS)
 
 PKG_NAME:=cypress-firmware
-PKG_VERSION:=v5.4.18-2020_0402
-PKG_RELEASE:=2
+PKG_VERSION:=v5.4.18-2020_0925
+PKG_RELEASE:=3
```

```suggestion
PKG_RELEASE:=1
```
请清空 commit message，并将 PKG_RELEASE 设置为 1